tests/sqllogictests/suites/query/02_function/02_0062_function_unnest

```diff
@@ -0,0 +1,97 @@
+query I
+select unnest([]::array(int));
+----
+
+query I
+select unnest([1,2,3]);
+----
+1
+2
+3
+
+query II
+select unnest([1,2,3]), number from numbers(1);
+----
+1 0
+2 0
+3 0
+
+query II
+select unnest([1,2,3]), number from numbers(2);
+----
+1 0
+2 0
+3 0
+1 1
+2 1
+3 1
+
+statement ok
+DROP DATABASE IF EXISTS db_02_0062;
+
+statement ok
+CREATE DATABASE IF NOT EXISTS db_02_0062;
+
+statement ok
+USE db_02_0062;
+
+statement ok
+create table t (a array(int));
+
+statement ok
+insert into t values ([1,2]), ([3,4,5]);
+
+query I
+select unnest(a) from t;
+----
+1
+2
+3
+4
+5
+
+query IT
+select unnest(a), a from t;
```

Comment:
so `select  3* (unnest(a) + 1) from t` is not supported now ?

Reply:
No, it's not supported now. It will be supported in the future.

`unnest` can only be the top of `ScalarExpr` now.